forem/forem
# Tags style

## What type of PR is this? (check all applicable)


## Description

This PR introduces some updates to tags styling:
- on feed no big change except flare tag will now be a bit less “in your face”
- on article view, we bring back colors - applying it to #  sign and also adding a bit decoration when hovering
- this basically diminish need for setting up two separate colors for admins (we currently do that for `bg` and `text` colors) - info about this is added to admin’s tag edit view as well as mod’s edit view.

More context:
- https://forem-team.slack.com/archives/C7GE84DEJ/p1633336128360100
- https://forem-team.slack.com/archives/C01MK34RE6P/p1635855371064200

## QA Instructions, Screenshots, Recordings

Click around wherever we have tags displayed (e.g. feed, search results, article view) and see if anything feels off.

https://user-images.githubusercontent.com/108287/139894226-0aac9ce4-cf8e-40cb-a524-50959eb14c32.mov

<img width="1427" alt="Zrzut ekranu 2021-11-2 o 13 14 47" src="https://user-images.githubusercontent.com/108287/139894435-75926b8d-fb31-4f91-b510-8e1dcf647a8c.png">

## Added/updated tests?


## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._
